mambaTests/HLSPlaylistStructureAndEditingTests.swift

```diff
@@ -937,3 +957,23 @@ let sample4SegmentPlaylist =
         "#EXTINF:2.002,\n" +
         "http://not.a.server.nowhere/segment4.ts\n" +
 "#EXT-X-ENDLIST\n"
+
```

Comment:
For new tests, can we use triple-quote strings to make these tests more readable, i.e.

```swift
let sampleDeltaUpdatePlaylist =
"""
#EXTM3U
#EXT-X-VERSION:9
#EXT-X-MEDIA-SEQUENCE:1
#EXT-X-SERVER-CONTROL:CAN-SKIP-UNTIL=12
#EXT-X-TARGETDURATION:2
#EXT-X-SKIP:SKIPPED-SEGMENTS=4
#EXTINF:2.002
http://not.a.server.nowhere/segment5.ts
#EXTINF:2.002
http://not.a.server.nowhere/segment6.ts
#EXTINF:2.002
http://not.a.server.nowhere/segment7.ts
#EXTINF:2.002
http://not.a.server.nowhere/segment8.ts
#EXTINF:2.002
http://not.a.server.nowhere/segment9.ts
#EXTINF:2.002
http://not.a.server.nowhere/segment10.ts
"""
```

Ran this through the test and it works

Reply:
Yes, that looks much more readable. Updated here: e6e31cafc326b3a5762d57515f25694814941361